public_html/app/Api/Responder.php

```diff
@@ -0,0 +1,24 @@
+<?php
```

Comment:
Responder should be responsible for whole response process. It should construct response. You should return responder, not plain API response.

Reply:
For example sometimes you want response to be in json format, sometimes in array. Responders should be responsible for this process.

Reply:
Your return could be something similar like this -
return new CitiesResponder($response);

and then you can process your response in that class with function toArray , toJson, isSuccess, isFailed etc...

Reply:
so you can check if your response is success easily $response->isSuccess(); or if you want to get content from response  $response->toArray(), if you want to return your response to another api $response->toJson();

Reply:
got it?